test/RunTestsOnWindows.cmd

```diff
@@ -0,0 +1,4 @@
+echo If it doesn't work, check if the version numbers in the paths are correct!
```

Comment:
I'm unsure about this file, to be honest... It seems unintended to me, but I could be wrong.

Reply:
As said above, there is no Cmake in QTCreator on Windows. And QtCreator does also not build these test snippets. This batch file runs qmake to perform the tests w/o me entering all path names on the command line. I can remove it, but it was meant as a helping hand for someone wanting to build an run the test snippets on Windows.  I have my .. well reservations .. about this test code anyway, because it only tests isolated library functions whereas most of the issues I found are in how things are working together in the global Mudlet framework. But Vadi insisted on the tests. To me, they are fun but only of little use to check the overall functioning of MXP within mudlet.

I can remove these helping scripts to compile the tests on Windows, no problem. But someother Windows builder might find them helpful.